examples/analyse_bonds.py

```diff
@@ -57,6 +57,28 @@ def test_analyse_comparing_bond_lengths():
     #print("maximium distance difference is :", difference[np.argmax(difference)])
     #print("minimum distance difference is :", difference[np.argmin(difference)])
 
+def test_analyse_chelation():
+    ## Initialises modules
+    from carmm.analyse.bonds import analyse_chelation
+    from ase import Atoms
+    import numpy as np
+    ## Builds a test hexahydrate complex to investigate coordination type.
+    p = np.array(
+        [[0, 0, 0],
+        [0, 0,  2.0], [0,  0.7,  2.7], [0, -0.7,  2.7],
+        [0, 0, -2.0], [0,  0.7, -2.7], [0, -0.7, -2.7],
+        [0,  2.0, 0], [0,  2.7,  0.7], [0,  2.7, -0.7],
+        [0, -2.0, 0], [0, -2.7, -0.7], [0, -2.7,  0.7],
+        [ 2.0, 0, 0], [ 2.7, -0.7, 0], [ 2.7,  0.7, 0],
+        [-2.0, 0, 0], [-2.7,  0.7, 0], [-2.7, -0.7, 0]])
+    atoms = Atoms('Mn(OH2)6', positions=p)
+    ## calculates chelation types
+    ligands = analyse_chelation(atoms=atoms, metal='Mn', ligand_atom='O', mult=1.5)
+    ## assertion check to verify results
+    assert(ligands.get("complex") == "Mn(κ1-H2O1)6")
```

Comment:
Not sure I quite get the meaning of k1 here? And the H2O1 is a touch confusing, as normally we'd drop the 1!

Reply:
κ1 is meant to represent the symbol for the denticity (ie: how many atoms in the ligand are coordinating to the metal center). So κ1 is monodentate, κ2 is bidentate ... 
Regarding the ligand formula, currently it counts the number of chemical symbols present using the counter function, but has no way of removing the one values. I'll try and look if there's a simple way of removing those one values.